views/main/play.ejs

```diff
@@ -31,6 +31,7 @@
         name="sessionId"
         value="<%=sessionId%>"
       />
+      <input type="hidden" id="imgLink" name="imgLink" value="<%=imgLink%>" />
```

Comment:
There already is a hidden input with the image url right underneath?

Reply:
Yep, that is some code that I accidentally left while I was trying to figure out what was the better approach to pixelate the images in `/play` pages. Both hidden inputs are actually unnecessary now, Thanks for pointing it out.